ui/components/app/snaps/snap-privacy-warning/snap-privacy-warning.js

```diff
@@ -1,8 +1,22 @@
 import React from 'react';
```

Comment:
I have noticed that there are several things that are different after the changes on this PR. I have compared this between the local build from this PR and the latest published flask (both running in Chrome on MAC OS).

This is screenshot of the current flask release from chrome app store:
![Screenshot 2023-08-24 at 20 41 51](https://github.com/MetaMask/metamask-extension/assets/13301024/c7c752dc-17f4-49b4-8c21-5d442bff3a20)

And this is from this PR:
![Screenshot 2023-08-24 at 20 43 55](https://github.com/MetaMask/metamask-extension/assets/13301024/38ad67c7-27c6-45a7-a3bc-b47e60a1d958)

I have noticed that scrollbars are now more emphasized somehow 🤔 (Scrollbars become visible when an external mouse is plugged into a Macbook, in my case)
I also noticed that the buttons are now not adjacent, but one above the other.
Also one more thing is that the link `Terms of Use` is now focused when this popup is opened, which was not the case before. I'm not sure why is it like that, but it looks a little bit unusual.

//cc @eriknson please review these findings and comment what are the requirements. Thanks!

Reply:
Ye, you're right, all of these (explicit scrollbar, focused links, and hidden scroll button) are regressions to me. Needs to be fixed before we merge this

Reply:
Updates to scrollbar, focused link and footer alignment have been made

<img width="361" alt="Screenshot 2023-08-28 at 5 46 56 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/e355c3cc-59d4-4a0b-8679-905174f53e50">